modules/persistent-storage-local-removing-devices.adoc

```diff
@@ -24,33 +19,72 @@ Deleting a persistent volume that is still in use can result in data loss or cor
 
 .Procedure
 
-. Edit the previously created local volume to remove any unwanted disks.
+To delete LVs or LVSs:
+
+. If there are any bound PVs, you need to delete the corresponding persistent volume claims (PVCs) to release the PVs:
+
+.. To find bound PVs, run the following command:
++
+[source, terminal]
+----
+$ oc get pv
+
+NAME   CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS      CLAIM                 STORAGECLASS   VOLUMEATTRIBUTESCLASS   REASON   AGE
+pv1    5Gi        RWO            Retain           Available                         slow           <unset>                          28s
+pv2    30Gi       RWX            Retain           Bound       openshift/storage     my-sc          <unset>                          168d
+----
++
+Bound PVs have a status of "Bound" and their corresponding PVCs appear in the "CLAIM" column. In the preceding example, the second PV is bound, and its PVC is "openshift/storage".
```

Comment:
```suggestion
Bound PVs have a status of `Bound` and their corresponding PVCs appear in the `CLAIM` column. In the preceding example, the second PV is bound, and its PVC is `openshift/storage`.
```
According to the OCP docs guidelines, I think these terms should be in backticks instead of double-quotation marks: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference. I suggested a few similar changes in other places in this file.